types/ember-changeset-validations/tsconfig.json

```
@@ -0,0 +1,27 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
```

```suggestion
            "es6"
```
why `dom` is needed?

---

seems like it's not needed, will remove it